rake_helper.rb

```diff
@@ -0,0 +1,130 @@
+require 'aws-sdk-lambda'
+require 'aws-sdk-cloudwatchevents'
+require 'yaml'
+
+class RakeHelper
+
+  attr_reader(
+    :travis_branch,
+    :aws_access_key_id,
+    :aws_secret_access_key,
+    :aws_configuration,
+    :lambda_client,
+    :yaml,
+    :lambda_config,
+    :function_name,
+    :event
+  )
+
+  def initialize
+    @travis_branch = ENV["TRAVIS_BRANCH"].upcase
+    @aws_access_key_id = ENV["AWS_ACCESS_KEY_ID_#{travis_branch}"]
+    @aws_secret_access_key = ENV["AWS_SECRET_ACCESS_KEY_#{travis_branch}"]
+    @aws_configuration = aws_configuration = {
+      region: 'us-east-1',
```

Comment:
Should this be set using an environment variable, or at least check if one is present and default to this if one isn't found? It will always be this value I think, but just to be safe.

Reply:
Changed.